specification/network/resource-manager/Microsoft.Network/stable/2024-10-01/networkVirtualAppliance.json

```diff
@@ -1764,6 +1783,88 @@
         }
       },
       "description": "Specifies input parameters required to retrieve the boot diagnostic logs for an NVA instance"
+    },
+    "NvaInterfaceConfigurations": {
+      "type": "array",
+      "readOnly": false,
+      "description": "The NVA in VNet interface configurations details",
+      "minItems": 2,
+      "maxItems": 3,
+      "items": {
+        "$ref": "#/definitions/NvaInterfaceConfigurationsProperties"
+      }
+    },
+    "NvaInterfaceConfigurationsProperties": {
+      "type": "object",
+      "properties": {
+        "subnet": {
+          "readOnly": false,
```

Comment:
>type [](http://example.com/codeflow?start=9&length=4)

[ARMBlockingComment] Don't use ARM top-level property names within the property bag. You could use nicType or interfaceType or something else that fits better.